webui/react/src/hooks/useKeyTracker.ts

```diff
@@ -64,4 +64,39 @@ const useKeyTracker = (): void => {
   }, [handleKeyUp, handleKeyDown]);
 };
 
+export type ShortcutConfig = {
+  altKey: boolean;
+  key: string;
+  metaKey: boolean;
+  shiftKey: boolean;
+  ctrlKey: boolean;
+};
+
+const keyNames = {
+  altKey: 'Alt',
+  ctrlKey: 'Ctrl',
+  metaKey: 'Super',
+  shiftKey: 'Shift',
+};
+
+export const shortcutMatch = (e: KeyboardEvent, shortcut: ShortcutConfig): boolean => {
+  return (
+    e.key.toUpperCase() === shortcut.key.toUpperCase() &&
+    (!shortcut.altKey || e.altKey) &&
+    (!shortcut.metaKey || e.metaKey) &&
+    (!shortcut.shiftKey || e.shiftKey) &&
+    (!shortcut.ctrlKey || e.ctrlKey)
+  );
+};
+
+export const getShortcutString = (shortcut: ShortcutConfig): string => {
+  let shortcutString = '';
+  if (shortcut.altKey) shortcutString += `${keyNames.altKey} +`;
```

Comment:
suggestion: instead of using `+`, empty space would be better since user might wanna use `+` key. Also, using `⌘`, `\u2303`, `⇧`, `⌥` is more intuitive than `Shift`, `Alt` etc

Reply:
Given that our shortcuts need to work on non-mac platforms, I don't think using those symbols is the right approach. Similarly, in the code you linked, there are three dicts for MacOS, Windows, and Linux.

Reply:
These are just examples for mac. I just thought its more intuitive but if its hard to configure symbols for all platforms, i'm fine with letter symbols